docs/guides/composer.mdx

```diff
@@ -0,0 +1,731 @@
+
+#### Copy and paste
+
+Click an operation and use the icons in the contextual menu to copy and paste it.
+
+#### Select multiple operations
+
+You can select several operations to copy and paste them, drag them to a new location, or group them into a custom unitary operation that displays in your operations catalog and functions as a single gate.
+
+To select more than one operation, place your cursor just outside one of the operations, then click and drag across the area to select. Shift-click individual operations to select or deselect them. A dashed line outlines the set of operations you are selecting, and each operation that is actually part of the selection is outlined in blue.
+
+For example, in the following image, the Hadamard gate on q1 and the CX gate are selected. The Hadamard gate on q0 is not selected.
+
+![A set of selected operations.](/docs/images/guides/composer/3c5c063d-b15f-43cb-bb65-9728187454d8.avif)
+
+Select Copy from the contextual menu to copy the group.
+
+To paste the group of operations, right-click in the circuit and select Paste.
+
+#### Build a custom operation using the group feature
+
+To group several operations together and save them as a custom operation, first select the operations as described above, then select Group from the contextual menu. You are prompted to name the custom operation or you can accept the default name. Click OK, and the custom operation will be represented by a single box, both in your circuit and in the operations catalog.
+
+You can now drag and drop the new operation throughout your circuit. Note that the operation is saved to this circuit but does not appear in the operations catalog for other circuits.
+
+You can also build a custom operation directly in the OpenQASM code editor; see [Create a custom operation in OpenQASM](#create-a-custom-operation-in-openqasm) for more information.
+
+#### Ungroup a custom or predefined operation
+
+To ungroup the gates within a custom or predefined operation, click the operation on the Composer and select Ungroup from the contextual menu. You can now move the separate operations individually. When you ungroup an operation, each element in the former group executes independently, which may mean they execute in a different order from when they were grouped together.
+
+#### Expand an operation’s definition
+
+To view the operations that constitute a custom or predefined operation without ungrouping them, click **Expand definition** from the contextual menu to see the defining gates. Click the icon again to collapse the definition.
+
+#### Rename or delete a custom operation
+
+To rename or delete a custom operation, right-click the operation in the operations catalog and select Rename or Delete. Deleting a custom operation from the operations catalog also deletes any instances of it on your circuit; however, deleting a custom operation from the circuit does not delete it from the operations catalog.
```

Comment:
I had to reread the second sentence a few times. (Granted, I'm fatigued right now hah). I think what trips me up is the `however` making me want to have a strong connection between the two clauses. Thoughts on splitting it out so the last clause is a standalone paragraph? 

>  To rename or delete a custom operation, right-click the operation in the operations catalog and select Rename or Delete. Deleting a custom operation from the operations catalog also deletes any instances of it on your circuit.

> Note that deleting a custom operation from the circuit itself does not remove it from the operations catalog.

Maybe even make the second sentence more verbose to really spell it out, like "and you must instead..."